test/fixtures/terminal/index.js

```diff
@@ -1,32 +1,19 @@
 /* global Pear */
-const { config,versions } = Pear
-const [grn, rst, dim] = ['\x1b[32m', '\x1b[0m', '\x1b[2m']
-const v = ({ key, length, fork }) => `v${fork}.${length}.${(key += '').length <= 12 ? key : key.slice(0, 12) + '…'}`
-const { app, platform } = await versions()
+const { teardown } = Pear
+import nodeInspector from 'bare-inspector'
```

Comment:
it's not `nodeInspector` though

Reply:
also your package.json has it mapped to `inspector` so either remove that from export map and import bare-inspector or change it to import inspector

Reply:
removed imports map in package.json since we are only running tests in bare. renamed import to `bareInspector`